public/van-latest.debug.js

```diff
@@ -1,4 +1,4 @@
-import van from "./van-latest.js"
+import van from "./van.js"
```

Comment:
a change here be expected?

Reply:
Hmm..., this is unexpected. This [line](https://github.com/vanjs-org/van/blob/main/src/publish.sh#L30) should have replaced the import file. Did you run `publish.sh` with bash, i.e.: `bash publish.sh`? Other shells might interpret the script differently. Also, did you notice any error message while running the script?

Reply:
@Tao-VanJS I had run `publish.sh` with bash on arch linux, and I saw an error on L1 

``sh
sed: -e expression #1, char 1: unknown command: `.'
``

maybe should I fix this manually?

Reply:
My development environment is Mac. Maybe the behavior is different. If you know a fix that can make it work in both environments, you can go ahead to fix it. Otherwise, if you change the script to:

```bash
if [[ "$(uname)" == "Linux" ]]; then
  sed ...
else
  sed ...
fi
```

Reply:
I just mean revert this change, I'm not good at bash script at all 😁

Reply:
That should work for this specific PR. The remaining steps after the `sed` are transpiling test files from `.ts` to `.js`. Since this PR doesn't change the test file, it should be fine.

I will probably merge this PR into a branch first, then do some manual testing, and finally merge it to `main`.